src/modeler/AutoRest.Swagger/Validation/ModelTypeIncomplete.cs

```diff
@@ -8,30 +8,23 @@
 
 namespace AutoRest.Swagger.Validation
 {
-    public class ModelTypeIncomplete : TypedRule<Schema>
+    public class ModelTypeIncomplete : DescriptionRequired<Schema>
     {
+        /// <summary>
+        /// Validates model for description property
+        /// </summary>
+        /// <param name="schema">Schema being validated</param>
+        /// <param name="context">Rule context</param>
+        /// <returns><c>true</c> if model contains description, <c>false</c> otherwise</returns>
         public override IEnumerable<ValidationMessage> GetValidationMessages(Schema schema, RuleContext context)
         {
-            if (schema != null && string.IsNullOrEmpty(schema.Reference) && schema.RepresentsCompositeType())
+            if (string.IsNullOrWhiteSpace(schema?.Reference) && schema.RepresentsCompositeType())
             {
                 if (schema.Description == null)
```

Comment:
should this condition also consider if description is empty or whitespace, similar to the other ones?